sdk/communication/communication-chat/src/signaling/signalingClient.browser.ts

```diff
@@ -12,8 +12,9 @@ export const getSignalingClient = (
   options?: SignalingClientOptions,
 ): SignalingClient | undefined => {
   return new CommunicationSignalingClient(credential, logger, {
-    environment: options?.environment ?? undefined,
     resourceEndpoint: options?.resourceEndpoint ?? undefined,
     gatewayApiVersion: options?.gatewayApiVersion ?? undefined,
+    additionalPolicies: options?.additionalPolicies ?? undefined,
```

Comment:
It's following existing code, but I don't think you need to `??` with `undefined`
```suggestion
    additionalPolicies: options?.additionalPolicies,
```